TSQL_Parser/TSQL_Parser/Clauses/TSQLOutputClause.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace TSQL.Clauses
+{
+	public class TSQLOutputClause : TSQLClause

```

Comment:
I do agree that OUTPUT is a clause, [OUTPUT Clause](https://docs.microsoft.com/en-us/sql/t-sql/queries/output-clause-transact-sql?view=sql-server-ver15) .